mapiproxy/servers/default/emsmdb/oxomsg.c

```diff
@@ -422,6 +422,7 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopTransportSend(TALLOC_CTX *mem_ctx,
 		DEBUG(0, ("Not implemented yet - shouldn't occur\n"));
 		break;
 	case true:
+		emsmdbp_object_attach_sharing_metadata_XML_file(emsmdbp_ctx, object);
```

Comment:
It is irrelevant for the subsequent calls. I could add it to have debugging lines added to the log, but I don't see its benefit.